terra-theme.config.js

```diff
@@ -0,0 +1,7 @@
+const themeConfig = {
+  exclude: [],
+  default: '',
```

Comment:
In the terra-toolkit doc, there doesn't appear to be a `default` option, should this be `theme` instead? https://github.com/cerner/terra-toolkit/blob/master/scripts/aggregate-themes/README.md#theme-optional

Reply:
updated to theme